charts/stable/ghost/values.yaml

```diff
@@ -0,0 +1,58 @@
+#
+# IMPORTANT NOTE
+#
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
+
+image:
+  # -- image repository
+  repository: ghost
+  # -- image tag
+  tag: 4.27.2
+  # -- image pull policy
+  pullPolicy: IfNotPresent
+
+# See https://ghost.org/docs/config/#running-ghost-with-config-env-variables
+env:
+  url: "http://some-ghost.example.com"
+  database__client: mysql
+  database__connection__host: mariadb
+  database__connection__user: ghost
+  database__connection__password: ghost
+  database__connection__database: ghost
```

Comment:
Since the app can be configured with sqlite, that might be a "better" default to provide. Then if users wish to use mysql, they can specify this themselves. Otherwise the chart will also not be able to pass the CI.

Reply:
Provided that the storage in each own cluster can be pretty much anything, sqlite is heavily frown upon default, but I suppose that it could be changed to default to it.

Reply:
It is also possible to stick with mysql, but then a different values file needs to be added for the CI to work. See for example the [Monica](https://github.com/k8s-at-home/charts/blob/master/charts/stable/monica/ci/ct-values.yaml) chart for an example